style.css

```diff
@@ -116,3 +123,260 @@ ul {
+}
+
+.content-project1 {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  height: 288px;
+  width: 295px;
+}
+
+.content-project1 h2 {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  margin-bottom: 0;
+  color: #172b4d;
+  height: 36px;
+  width: 295px;
+}
+
+.content-project1 .ali {
+  display: flex;
+  height: 32px;
+  width: 236px;
+  gap: 12px;
+}
+
+.content-project1 .ali .ali1 {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+  width: 55px;
+  height: 16px;
+  flex: 1 1 40%;
+}
+
+.content-project1 .ali .det {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  flex: 1 1 90%;
+}
+
+.content-project1 .parg {
+  width: 295px;
+  height: 76px;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.content-project1 .lan {
+  display: flex;
+  flex-direction: row;
+  padding: 4px 12px;
+}
+
+.sort .lang {
+  width: 327px;
+  height: 24px;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  color: #000;
+  margin-bottom: 36px;
+}
+
+.content-project1 .lan .lang {
+  display: flex;
+  flex-direction: row;
+  justify-content: left;
+  background-color: #ebebff;
+  border-radius: 8px;
+  padding: 4px 12px;
+  margin-right: 8px;
+  margin-top: 12px;
+  text-align: center;
+  gap: 8px;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
+.see-p .see-project {
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  width: 140px;
+  height: 48px;
+  background: #fff;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  color: #396df2;
+}
+
+.first-project .snapshot1 {
+  width: 295px;
+  height: 220px;
+  background: url("images/project2.png");
+  border-radius: 8px;
+}
+
+.first-project .snapshot2 {
+  width: 295px;
+  height: 220px;
+  background: url("images/project3.png");
+  border-radius: 8px;
+}
+
+.first-project .snapshot3 {
+  width: 295px;
+  height: 220px;
+  background: url("images/project4.png");
+  border-radius: 8px;
+}
+
+.about-me {
+  width: 375px;
+  height: 1263px;
+  background: #fff;
+  display: flex;
+  padding: 114px 24px;
+  flex-wrap: wrap;
+  margin: 0;
+}
+
+.about-me .about {
+  display: flex;
+  flex-wrap: wrap;
+  width: 327px;
+  height: 409px;
+}
+
+.about-me .about h2 {
+  width: 327px;
+  height: 101px;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  margin-top: 0;
+  margin-bottom: 0;
+}
```

Comment:
- [ ] [REQUIRED] Please add this styles `
width: 327px;
height: 101px;
font-family: 'Poppins';
font-style: normal;
font-weight: 700;
font-size: 40px;
line-height: 52px;
display: flex;
align-items: center;
color: #172B4D;`! 

**The picture on the left is yours**

![image](https://user-images.githubusercontent.com/63915024/176978214-9bdec092-6ddc-4075-bf14-a3b4498a6c24.png)